packages/main/src/types/TabLayout.js

```diff
@@ -0,0 +1,40 @@
+import DataType from "@ui5/webcomponents-base/dist/types/DataType.js";
+
+/**
+ * @lends sap.ui.webcomponents.main.types.TabDesign.prototype
```

Comment:
TabLayout or the docs will not be built correctly

Reply:
agree, first I called it Design, then Layout